homeassistant/components/ambiclimate/__init__.py

```diff
@@ -20,7 +20,7 @@
 )
 
 
-async def async_setup(hass, config):
+async def async_setup(hass, config) -> bool:
```

Comment:
Please try to avoid partial typing, instead, when typing method type it at full.